Index: qa/py/qa/util/agave_tools/volume_group_acropolis_util.py
diff --git a/qa/py/qa/util/agave_tools/volume_group_acropolis_util.py b/qa/py/qa/util/agave_tools/volume_group_acropolis_util.py
index 7400961a616492ab2d1cb7512a5cbe0bfa4881f3..eaaeb33950479fc8e86589dad362ca62ae0b04f1 100644
--- a/qa/py/qa/util/agave_tools/volume_group_acropolis_util.py
+++ b/qa/py/qa/util/agave_tools/volume_group_acropolis_util.py
@@ -19,15 +19,16 @@ class VolumeGroupAcropolisUtil(object):
   """
 
   @staticmethod
-  def create_volume_group(svm, volume_group_name):
+  def create_volume_group(svm, volume_group_name,shared=False):
     """
     Creates a volume group with the specified name at the specified svm.
     svm: SVM where the ACLI command will be executed to create the volume
          group.
     volume_group_name: Name to be assigned to the volume group.
+    shared: True if shared volume group.
     Returns (True, Data) on success, (False, Error Message) on failure.
     """
-    vg_create_cmd = "vg.create %s" % volume_group_name
+    vg_create_cmd = "vg.create %s shared=%s" % (volume_group_name, shared)
     ret, stdout, stderr = VolumeGroupAcropolisUtil.__execute_acli(
         svm, vg_create_cmd)
     return VolumeGroupAcropolisUtil.__parse_output(
